docs/release-notes/ngc-images.rst

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+**New Features**
+
+-  Add experimental NGC-based images in our environments offerings. These images can be pulled from
+   `https://hub.docker.com/r/determinedai/pytorch-ngc` or
+   `https://hub.docker.com/r/determinedai/tensorflow-ngc`
+
+**Improvements**
+
+-  Remove TF 2.8 images from our offerings. Our default TF 2.11 images are still available for
+   Tensorflow users.
```

Comment:
**New Features**

-  Add experimental NGC-based images in our environments offerings. These images can be pulled from
   `https://hub.docker.com/r/determinedai/pytorch-ngc` or
   `https://hub.docker.com/r/determinedai/tensorflow-ngc`

**Improvements**

-  Remove TF 2.8 images from our offerings. Default TF 2.11 images are still available for TensorFlow users.